src/types/ErrorRate.ts

```diff
@@ -0,0 +1,171 @@
+import { NA, RequestHealth } from './Health';
+import { RateHealthConfig, ToleranceConfig } from './ServerConfig';
+import { serverConfig } from '../config/ServerConfig';
+import { ThresholdStatus, getRequestErrorsStatus, RATIO_NA } from './Health';
+
+export interface ErrorRatio {
+  errorRatio: { status: ThresholdStatus; protocol?: string };
+  inboundErrorRatio: { status: ThresholdStatus; protocol?: string };
+  outboundErrorRatio: { status: ThresholdStatus; protocol?: string };
+}
+
+export interface Rate {
+  requestRate: number;
+  errorRate: number;
+  errorRatio: number;
+}
+
+export const regexProtocol = {
+  grpc: new RegExp(/^[1-9]$|^1[0-6]$/),
+  /* eslint-disable-next-line */
```

Comment:
why eslint disabled here?

Reply:
good question ^^ Do you remenber the conversation about the grpc regex where I forgot the slash in \d\d ?? Tachan !!! eslint is removing this slash, I don't know why.

Reply:
weird, but I just tried locally (after re-enabled eslint) and, running `yarn lint`, it doesn't change anything.
and we're supposed to all have the same version of the linter...